applications/GeoMechanicsApplication/tests/test_dsettlement_validation.py

```diff
@@ -1,9 +1,140 @@
 from KratosMultiphysics.GeoMechanicsApplication import run_multiple_stages
+from KratosMultiphysics.GeoMechanicsApplication import unit_conversions
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import os
+import pathlib
 
 import test_helper
 
+if test_helper.want_test_plots():
+    import KratosMultiphysics.GeoMechanicsApplication.geo_plot_utilities as plot_utils
+
+
+def _extract_x_and_y_from_line(line, index_of_x=0, index_of_y=1):
```

Comment:
Since this function isn't in a class, should the _ be removed at the start?
```suggestion
def extract_x_and_y_from_line(line, index_of_x=0, index_of_y=1):
```

Reply:
I'm in the habit of prepending "private" function names with an underscore. But perhaps we should have a discussion with the team about this, to make sure we all use the same convention. Thanks for bringing this up.

Reply:
Yes, as discussed, I wasn't used to this concept of 'private free functions', but it makes sense (especially since we don't have things like anonymous namespaces to make these functions private if we'd like to in python.